atom/browser/api/atom_api_window.cc

```diff
@@ -83,7 +83,7 @@ Window::Window(v8::Isolate* isolate, v8::Local<v8::Object> wrapper,
   v8::Local<v8::Value> transparent;
   if (options.Get("transparent", &transparent))
     web_preferences.Set("transparent", transparent);
-
```

Comment:
Thanks. But the error message is strange. And i think add the white space is more readable. Any way, i removed the whitespace.